### DISCUSSION TITLE
Source code

### DISCUSSION THREAD

**Discussion by ghost** [AzuraCast/AzuraCast#5749] [Q&A]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release

### What happened?

How can i modify source code of azuracast installed using docker

**Comment by Vaalyn** [comment]
We do not give support for modifying the source code of an installation.

Any direct modification to the source code of an installation will be overwritten on the next update so doing this is rather futile and will in most cases lead to frustration.
